core/common/src/main/java/alluxio/util/io/FileUtils.java

```diff
@@ -42,6 +48,23 @@
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
 
   /**
+   * Changes the local file's group.
+   *
+   * @param path that will change owner
+   * @param group the new group
+   * @throws IOException if the group is unable to be changed
+   */
+  public static void changeLocalFileGroup(String path, String group) throws IOException {
```

Comment:
I couldn't find a good way to do so because there's no guarantee of what groups / users are available in different environments. Do you have any suggestions?